eng/pipelines/mono/templates/build-job.yml

```diff
@@ -13,6 +13,9 @@ parameters:
   isOfficialBuild: false
   crossBuild: false
   crossrootfsDir: ''
+  dependOnEvaluatePaths: false
```

Comment:
This is outdated, I updated this to: `dependOnCheckoutBundle` here: https://github.com/dotnet/runtime/commit/604660b37a0d416155a97cbd09c1edc101a1a1cb